### PR TITLE
Replaced datetime.now().timestamp() with time.time()

### PR DESCRIPTION
**Pull request by Minkowski104**

### CODE REVIEW

**Review by stefanluth — COMMENTED** [review]
Hi, thank you for your help! There are two more occurrences of `datetime.now().timestamp()` in `time_tracker.py`. You should then be able to remove the datetime import within that file.

File: modules/time_tracker/ProfilesDB.py [hunk 1]
```diff
@@ -1,5 +1,6 @@
 from datetime import datetime
```
Comment (stefanluth) [inline modules/time_tracker/ProfilesDB.py:1]:
```suggestion
```
This import can be removed as well.

**Review by stefanluth — CHANGES_REQUESTED** [review]

**Comment by stefanluth** [comment]
Please create a new branch for this pull request. If you need help on how to do that, just let me know :) A few guidelines can be found [here](https://git-scm.com/book/en/v2/Git-Branching-Basic-Branching-and-Merging) and [here](https://github.com/Kunena/Kunena-Forum/wiki/Create-a-new-branch-with-git-and-manage-branches).

**Comment by Minkowski104** [comment]
> Please create a new branch for this pull request. If you need help on how to do that, just let me know :) A few guidelines can be found [here](https://git-scm.com/book/en/v2/Git-Branching-Basic-Branching-and-Merging) and [here](https://github.com/Kunena/Kunena-Forum/wiki/Create-a-new-branch-with-git-and-manage-branches).

I have create a new branch by the name b1 for the pull request. #29  I hope I have done it right finally. I really appreciate your guidance. Please let me know if I have done it right this time.

**Comment by stefanluth** [comment]
> I have create a new branch by the name b1 for the pull request. #29 I hope I have done it right finally. I really appreciate your guidance. Please let me know if I have done it right this time.

Looks good! I'll continue our conversation over there :)

**Pull request closed without merging**